docs/code/MlNetCookBook.md

```diff
@@ -41,6 +41,7 @@ Please feel free to search this page and use any code that suits your needs.
 - [How do I train using cross-validation?](#how-do-i-train-using-cross-validation)
 - [Can I mix and match static and dynamic pipelines?](#can-i-mix-and-match-static-and-dynamic-pipelines)
 - [How can I define my own transformation of data?](#how-can-i-define-my-own-transformation-of-data)
+- [How can I read and write binary data?](#how-can-i-read-and-write-binary-data)
```

Comment:
Would this make sense to be next to the "How do I load data from a text file?" question?

Reply:
Yes I agree! :) 

---
In reply to: [263541837](https://github.com/dotnet/machinelearning/pull/2811#discussion_r263541837) [](ancestors = 263541837)